src/preferences/dialog/dlgprefvinyldlg.ui

```diff
@@ -424,183 +543,55 @@
-            <verstretch>0</verstretch>
-           </sizepolicy>
-          </property>
-          <property name="sizeHint" stdset="0">
-           <size>
-            <width>30</width>
-            <height>20</height>
-           </size>
-          </property>
-         </spacer>
-        </item>
-
-        <item>
-         <widget class="QLabel" name="textLabelPreampCurrent">
-          <property name="text">
-           <string notr="true">TextLabel</string>
-          </property>
-         </widget>
-        </item>
-
-        <item>
-         <spacer name="horizontalSpacer_3">
-          <property name="orientation">
-           <enum>Qt::Horizontal</enum>
-          </property>
-          <property name="sizePolicy">
-           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
-            <horstretch>0</horstretch>
-            <verstretch>0</verstretch>
-           </sizepolicy>
-          </property>
-          <property name="sizeHint" stdset="0">
-           <size>
-            <width>30</width>
-            <height>20</height>
-           </size>
-          </property>
-         </spacer>
-        </item>
-
-        <item>
-         <widget class="QLabel" name="textLabelPreampMax">
-          <property name="sizePolicy">
-           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
-            <horstretch>0</horstretch>
-            <verstretch>0</verstretch>
-           </sizepolicy>
-          </property>
-          <property name="text">
-           <string>44 dB</string>
-          </property>
-          <property name="wordWrap">
-           <bool>false</bool>
-          </property>
-         </widget>
-        </item>
-
-       </layout>
-      </item>
-     </layout>
-    </widget>
-   </item>
-
-   <item row="2" column="0" rowspan="2">
+   <item row="2" column="0">
     <widget class="QGroupBox" name="groupBoxSignalQuality">
      <property name="title">
       <string>Signal Quality</string>
      </property>
      <property name="alignment">
-      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
+      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
+     </property>
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
      </property>
      <layout class="QGridLayout">
-      <item row="0" column="0">
-       <spacer name="verticalSpacer_2">
-        <property name="orientation">
-         <enum>Qt::Vertical</enum>
-        </property>
-        <property name="sizeHint" stdset="0">
-         <size>
-          <width>20</width>
-          <height>40</height>
-         </size>
-        </property>
-       </spacer>
-      </item>
      </layout>
     </widget>
    </item>
 
-   <item row="5" column="0" colspan="2">
-    <widget class="QLabel" name="label">
-     <property name="enabled">
-      <bool>false</bool>
+   <item row="3" column="0" colspan="2">
```

Comment:
this is the relevant part, the other changes are just re-ordering grid items according to actual order in UI.